code/dallus_content/items/clothing/donator/ties.dm

```diff
@@ -6,6 +6,34 @@
 	icon_state = "watch"
 	item_state = "watch"
 	item_color = "watch"
+	var/cooldown_timer = 300
+
+/obj/item/clothing/accessory/watch/attack_self(mob/usr)
+	var/mob/living/carbon/human/H = usr
```

Comment:
You should run a sanity check here to make sure your user is actually a human before you assign them to H.